examples/chrome-extension/src/popup.ts

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+// This code is partially adapted from the openai-chatgpt-chrome-extension repo:
+// https://github.com/jessedi0n/openai-chatgpt-chrome-extension
+
+import {ChatRestModule} from "@mlc-ai/web-llm";
+import './popup.css';
+
+// Load chat module
+const cm = new ChatRestModule();
```

Comment:
Note: I got around build errors by importing `@mlc-ai/web-llm` and instantiating `ChatRestModule` in the main content script `popup.ts` instead of a background worker (which was throwing an error about imports). This may not be the best design, since we are instantiating a new `ChatRestModule` every time the user clicks on the extension icon. Ideally the interfacing with WebLLM would happen in the background worker, but at this point I am unsure of how to do that successfully.